src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagController.cs

```diff
@@ -341,7 +341,7 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
             [FromQuery][Range(-1, int.MaxValue)]
             int durationSeconds = 30,
             [FromQuery]
-            LogLevel level = LogLevel.Debug,
+            LogLevel level = LogLevel.Warning,
```

Comment:
Maybe we could make `level` nullable, so that we could use `settings.LogLevel = level.GetValueOrDefault(LogLevel.Warning)`